modules/local/starfusion/build/main.nf

```diff
@@ -1,37 +1,32 @@
 process STARFUSION_BUILD {
-    tag 'star-fusion'
+    tag "$meta.id"
+    label 'process_high'
 
     conda "${moduleDir}/environment.yml"
-    container 'community.wave.seqera.io/library/dfam_hmmer_samtools_star-fusion_pruned:5694d82381bf039e'
+    container 'community.wave.seqera.io/library/dfam_hmmer_minimap2_samtools_pruned:63e3d21ca68ea531'
 
     input:
     tuple val(meta), path(fasta)
     tuple val(meta2), path(gtf)
+    path fusion_annot_lib
+    val dfam_species
 
     output:
-    path "*"  , emit: reference
+    path "ctat_genome_lib_build_dir"  , emit: reference
```

Comment:
Make corresponding changes on the publishing directory in modules.config so the reference folder structure remains the same